backstopper-jaxrs/build.gradle

```diff
@@ -0,0 +1,19 @@
+evaluationDependsOn(':')
+
+dependencies {
+    compile(
+            project(":backstopper-servlet-api"),
+            project(":backstopper-jackson"),
+            "javax.ws.rs:javax.ws.rs-api:2.0.1",
+    )
+    testCompile(
+            project(":backstopper-core").sourceSets.test.output,
+            "junit:junit:$junitVersion",
+            "org.mockito:mockito-core:$mockitoVersion",
+            "ch.qos.logback:logback-classic:$logbackVersion",
+            "org.assertj:assertj-core:$assertJVersion",
+            "com.tngtech.java:junit-dataprovider:$junitDataproviderVersion",
+            "org.hamcrest:hamcrest-all:$hamcrestVersion",
+            "org.glassfish.jersey.core:jersey-server:$jersey2Version",
```

Comment:
Is this jersey-server dependency still needed for the tests or can you remove it?